site/content/en/docs/Installation/Terraform/eks.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Installing Agones on AWS Elastic Kubernetes Service using Terraform"
+linkTitle: "AWS"
+weight: 20
+description: >
+  You can use Terraform to provision an EKS cluster and install Agones on it.
+---
+
+{{% feature expiryVersion="1.3.0" %}}
+This feature is under development and will be supported in the upcoming release.
+{{% /feature %}}
+
+{{% feature publishVersion="1.3.0" %}}
+## Installation
+
+### Creating Cluster
+
+You can use Terraform to provision your Amazon EKS (Elastic Kubernetes Service) cluster and install Agones on it using the Helm Terraform provider.
+
+The example of the EKS submodule config file can be found here:
+ {{< ghlink href="examples/terraform-submodules/eks/module.tf" >}}Terraform configuration with Agones submodule{{< /ghlink >}}
+
+Copy `module.tf` file into a separate folder.
+
+Configure your AWS CLI tool [CLI configure](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html):
+```
+aws configure
+```
+
+Configure your terraform:
+```
+terraform init
+```
+
+By editing `modules.tf` you can change the parameters that you need to, such as the EC2 instance type. Note that the maximum number of instances in the workers group is limited to 3.
```

Comment:
> Note that the maximum number of instances in the workers group is limited to 3.

Is there some docs or something we can point this to? It sounds like EKS can only have 3 nodes in the cluster, which I'm assuming is not correct?

I did some searching, but I couldn't find a clear example (to me) or what a worker group is?

Reply:
This limitation could be found when I change NodePool to 4 I got an error message
 I will point to some proof or a ticket here.

Reply:
Trying to find a link to the proof, submitted a version with fixing all other comments for now.

Reply:
This statement above was incorrect, solved by adding `asg_max_size=4`:
```
Error: Error creating AutoScaling Group: ValidationError: Desired capacity:4 must be between the specified min size:1 and max size:3
        status code: 400, request id: c0e617ab-3218-11ea-b15b-974086fb280d
```